ui/src/app/shared/components/badge-panel/badge-panel.tsx

```diff
@@ -9,66 +9,67 @@ require('./badge-panel.scss');
 export const BadgePanel = ({app, project}: {app?: string; project?: string}) => {
     const [badgeType, setBadgeType] = React.useState('URL');
     const context = React.useContext(Context);
-    const root = `${location.protocol}//${location.host}${context.baseHref}`;
-
-    let badgeURL = '';
-    let entityURL = '';
-    let alt = '';
-    if (app) {
-        badgeURL = `${root}api/badge?name=${app}&revision=true`;
-        entityURL = `${root}applications/${app}`;
-        alt = 'App Status';
-    } else if (project) {
-        badgeURL = `${root}api/badge?project=${project}&revision=true`;
-        entityURL = `${root}projects/${project}`;
-        alt = 'Project Status';
-    } else {
+    if (!app && !project) {
         throw new Error('Either app of project property must be specified');
```

Comment:
I know this is not part of your change, but I just spotted this little typo.

```suggestion
        throw new Error('Either app or project property must be specified');
```